umpire-config.cmake.in

```diff
@@ -9,7 +9,7 @@ set(UMPIRE_INCLUDE_DIRS "@CMAKE_INSTALL_PREFIX@/include")
 
 if (NOT TARGET camp)
   if (NOT DEFINED camp_DIR)
-    set(camp_DIR @CMAKE_INSTALL_PREFIX@/lib/cmake/camp)
+    set(camp_DIR @camp_DIR@)
```

Comment:
that should do it, i usually use prefixed names for things I save, but thats just an extra precaution.